implementation/inferius/src/main/java/com/example/inferius/shared/UUIDv7Validator.java

```diff
@@ -0,0 +1,29 @@
+package com.example.inferius.shared;
```

Comment:
```suggestion
package pwr.jakprzyjade.inferius.shared;
```

Not sure how easy it is to change now.

Reply:
Not very difficult, done